tablepyxl/tablepyxl.py

```diff
@@ -28,25 +28,33 @@ def write_rows(worksheet, elem, row, column=1):
 
     returns the next row after all rows are written
     """
+    from openpyxl.cell.cell import MergedCell
+
     initial_column = column
     for table_row in elem.rows:
         for table_cell in table_row.cells:
+            cell = worksheet.cell(row=row, column=column)
+            while isinstance(cell, MergedCell):
+                column += 1
+                cell = worksheet.cell(row=row, column=column)
+
             colspan = string_to_int(table_cell.element.get("colspan", "1"))
             rowspan = string_to_int(table_cell.element.get("rowspan", "1"))
             if rowspan > 1 or colspan > 1:
                 worksheet.merge_cells(start_row=row, start_column=column,
                                       end_row=row + rowspan - 1, end_column=column + colspan - 1)
-            cell = worksheet.cell(row=row, column=column)
```

Comment:
It looks like the error might have been here. What happens if we do `column=column + colspan - 1` on this line instead of the other change?

Reply:
`column=column + colspan - 1` can't fix this error:
```python
from tablepyxl import tablepyxl

table = """<table>
  <tr>
    <th colspan="2" rowspan="2">a1</th>
    <th colspan="2">c1</th>
  </tr>
  <tr>
    <td rowspan="2">c2</td>
    <td>d2</td>
  </tr>
  <tr>
    <td>a3</td>
    <td>b3</td>
    <td>d3</td>
  </tr>
</table>"""

tablepyxl.document_to_xl(table, "test.xlsx")
# AttributeError: 'MergedCell' object has no attribute 'value'
```


![table](http://i2.tiimg.com/582188/9674201d0878622b.png)

we can add this code before `cell = worksheet.cell(row=row, column=column)` to analyse this bug: 
```python
print(f'We think we are in row={row}, column={column},',
      f'And now actually in {table_cell.value}, ', table_cell.element.attrib)
```
```text
We think we are in row=1, column=1, And now actually in a1,  {'colspan': '2', 'rowspan': '2'}
We think we are in row=1, column=3, And now actually in c1,  {'colspan': '2'}
We think we are in row=2, column=1, And now actually in c2,  {'rowspan': '2'}
raise AttributeError: 'MergedCell' object has no attribute 'value'
```

In row=2, we think we are start from column=1, but actually A2 and B2 were already builded from "a1", so I judge MergedCell to skip them and shift to the actually column=3.